lib/block-list.js

```diff
@@ -32,5 +34,25 @@ function renderItem (item) {
 }
 
 export default function ({body}) {
-  return body.map(renderItem).filter(Boolean);
+  const bundlesToUrls = {};
+  const urlsToBundles = {};
+  let count = 0;
+
+  const components = body
+    .map(renderItem)
+    .filter(Boolean);
+
+  components.forEach(item => {
+    if (item.components && item.components[0] && item.components[0].role === 'photo') {
+      const url = item.components[0].URL;
+      if (!urlsToBundles[url]) {
+        urlsToBundles[url] = `image-${count}`;
+        bundlesToUrls[`image-${count}`] = url;
```

Comment:
ok, nvm, https://www.npmjs.com/package/file-type will handle this on apple-news uploader side